configure

```diff
@@ -664,7 +664,7 @@ def configure_v8(o):
   if options.shared_v8_libname:
     o['libraries'] += ['-l%s' % options.shared_v8_libname]
   elif options.shared_v8:
-    o['libraries'] += ['-lv8']
+    o['libraries'] += ['-lv8', '-lv8_platform']
```

Comment:
@bnoordhuis won't this be an issue when providing your own shared libname (since you can only provide one)? I guess it's irrelevant when my shared v8 patch lands..